uswds/uswds
# Merge master back into develop - Release 1.6.7

* Adding best-practice ruleset and  exceptions to axe test script. Updated heading level to fix axe best-practice error.

* Update README.md

* Link to scripts in line 28

* Add new sites to Showcase list

* Fix order

* Update WHO_IS_USING_USWDS.md

* Update WHO_IS_USING_USWDS.md

* Add new sites

* Changed URL for login.gov

* [ci] test filtering out pull requests from ci build

* [ci] moving branch filter to only stop codeclimate

* [ci] testing regex in bash

* [app] Update eslint to use airbnb style guide

* [app] beginning to lint all test files, update gulp-eslint

* [app] update more files according to eslint defaults

* [app] updates sass spec files

* [app] fix tests

* Use named function in many cases to ensure proper scope for `this`

* [app] lint all the things

* [app] Begin breaking out focus trap using receptor

* [app] use /util/focus-trap in navigation component

* [app] lint fixes

* [docs] Update README to reflect actual npm package folder structure

remove non-essential files from example

* Add site

* [app] fix code climate complexity warnings

* Fix radio and checkbox left alignment (#2593)

* Move checkbox CSS away from combined radio code

* Adjust left margin for proper alignment

* Add sites

* [UIComponent] use element.hidden on search form

* Previously, we hid the search form in the extended header using a
screen reader only class. This led to users being able to tab through
the invisible form fields, which was confusing

* Update package-lock.json

* Tidy webkit dob input styles

* Remove outer spin button bc not in shadow DOM

* Remove margin rule bc didn't do anything

* Add appearance none to safari profile shadow dom

* Wrap in a form element to allow autocomplete to show up

* Try a stronger autocomplete hide solution

* Disable stylelint declaration-no-important rule

* Remove unneeded grid divs from search component (#2607)

* Remove unneeded divs from search code sample

* Update package-lock.json

* 1.6.7



## Description

Include a high-level description of the issue your pull request fixes; include only one issue per pull request. Please also link to the GitHub issue it resolves.

## Additional information

Include any of the following (as necessary): 

* Relevant research and support documents
* Type of content review needed: stylistic or copy editing
* Screenshot images
* Notes

Before you hit Submit, make sure you’ve done whichever of these applies to you:
